texttospeech/test/quickstart.test.mjs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import fs from 'fs/promises';
+import { existsSync } from 'node:fs';
+import { describe, it } from 'mocha';
+import * as cp from 'child_process';
+import { assert } from 'chai';
+
+const outputFile = 'quickstart_output.mp3';
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
```

Comment:
question: Not sure if we have a pattern on this, but since we are using only `execSync` from `child_process`, should we follow whichever pattern is used by fs/promises on including the library or the one function to use?

Also, since we only calling `execSync` once, might it be simpler to directly add the encoding directive inline and skip the setup?